### PR TITLE
Bug 1186146 - [gallery/music] Hide dotfiles in MediaDB

### PR DESCRIPTION
**Pull request by jimporter**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=dd92a3f6f7f6348b511f8f7710d5875aff129baf

File: shared/js/mediadb.js [hunk 1]
```diff
@@ -1420,14 +1420,11 @@ var MediaDB = (function() {
   // Test whether this filename is one we ignore.
   // This is a separate function because device storage change events
   // give us a name only, not the file object.
-  // Ignore files having directories beginning with .
```
Comment (davidflanagan) [inline shared/js/mediadb.js]: Instead of just deleting this line, would you update it, please?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=240c7aabe4c1cde095f65b3067a0238ee4924d39

File: shared/js/mediadb.js (hunk 1 above)
Comment (jimporter) [inline shared/js/mediadb.js]: Done.

**Pull request merged**